src/OrchardCore.Modules/OrchardCore.Contents/Drivers/OwnerEditorDriver.cs

```diff
@@ -70,7 +78,16 @@ public class OwnerEditorDriver : ContentPartDisplayDriver<CommonPart>
 
                 if (!string.IsNullOrEmpty(part.ContentItem.Owner) || !context.IsNew)
                 {
-                    part.ContentItem.Owner = model.Owner;
+                    var newOwner = await _userManager.FindByNameAsync(model.Owner);
```

Comment:
Add security checks

https://github.com/OrchardCMS/Orchard/blob/dev/src/Orchard.Web/Core/Common/OwnerEditor/OwnerEditorDriver.cs#L37

Reply:
There is already that kind of check:

            if (currentUser == null || !(await _authorizationService.AuthorizeAsync(currentUser, StandardPermissions.SiteOwner, part)))
            {
                return null;
            }